temporalio/testing/_activity.py

```diff
@@ -74,15 +74,30 @@ def __init__(self) -> None:
         self._cancelled = False
         self._worker_shutdown = False
         self._activities: Set[_Activity] = set()
+        self._cancellation_details = (
+            temporalio.activity._ActivityCancellationDetailsHolder()
+        )
 
-    def cancel(self) -> None:
+    def cancel(
+        self,
+        cancellation_details: Optional[
+            temporalio.activity.ActivityCancellationDetails
+        ] = None,
```

Comment:
Updated https://github.com/temporalio/sdk-python/pull/854#discussion_r2076164757 thread, I think this parameter should not be `Optional`

Reply:
Didn't mean to resolve - param is non-optional. Updated callers.